### PR TITLE
Extracted checkbox code from kiwi-mobile(link in description)

### PR DESCRIPTION
**Pull request by denbalogh**
https://github.com/kiwicom/mobile/blob/master/packages/shared/src/forms/Checkbox.js

### CODE REVIEW

**Comment by RobinCsl** [comment]
Please, rename the issue to be more specific with what code you extracted. Thank you!

File: storybooknative/src/Checkbox/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import styled from "../styled";
+import Touchable from "../Touchable";
+
+import Props from "./index";
+
+export default function Checkbox(props: Props) {
+  const StyledIconPlaceholder = styled(View, () => ({}));
+
+  const StyledWrapper = styled(View, () => ({}));
```
Comment (tbergq) [inline storybooknative/src/Checkbox/index.js]: Why are this styled wrapper without any styles used instead of just a `View`?

Comment (tbergq) [reply storybooknative/src/Checkbox/index.js]: Also if the intent is to add styles here later, the definition should still be outside of the component. Now it is being recreated on each render.

Comment (denbalogh) [reply storybooknative/src/Checkbox/index.js]: Fixed the definition scope and yes, it is just a set up for next iteration where I am adding the styles.

**Pull request merged**